apps/app/dashboards/public-safety/fire-and-rescue/index.tsx

```diff
@@ -1,37 +1,246 @@
+                  })}
+                  enableAnimation={!play}
+                  interval="day"
+                  data={{
+                    labels: coordinate.x,
+                    datasets: [
+                      {
+                        type: "line",
+                        data: coordinate.overall,
+                        label: t("daily"),
+                        borderColor: AKSARA_COLOR.DANGER,
+                        borderWidth: 1.5,
+                        backgroundColor: AKSARA_COLOR.DANGER_H,
+                        fill: true,
+                      },
+                    ],
+                  }}
+                  stats={[
+                    {
+                      title: t("daily"),
+                      value: `+${numFormat(
+                        timeseries_callout.data.data[currentState].overall.daily.value,
+                        "standard"
+                      )}`,
+                    },
+                    {
+                      title: t("total"),
+                      value: numFormat(
+                        timeseries_callout.data.data[currentState].overall.cumul.value,
+                        "standard"
+                      ),
+                    },
+                  ]}
+                />
+                <Slider
+                  type="range"
+                  value={data.minmax}
+                  data={timeseries.data.x}
+                  onChange={e => setData("minmax", e)}
+                />
+              </>
+            )}
+          </SliderProvider>
+          <div className="grid grid-cols-1 gap-12 pt-8 lg:grid-cols-3">
+            {OPERATION.map((key: string) => (
+              <Timeseries
+                key={key}
+                title={t(key)}
+                className="h-[300px] w-full"
+                interval={"day"}
+                data={{
+                  labels: coordinate.x,
+                  datasets: [
+                    {
+                      type: "line",
+                      data: coordinate[key],
+                      label: t("daily"),
+                      borderColor: AKSARA_COLOR.DANGER,
+                      borderWidth: 1.5,
+                      backgroundColor: AKSARA_COLOR.DANGER_H,
+                      fill: true,
+                    },
+                  ],
+                }}
+                stats={[
+                  {
+                    title: t("daily"),
+                    value: `+${numFormat(
+                      timeseries_callout.data.data[currentState][key].daily.value,
+                      "standard"
+                    )}`,
+                  },
+                  {
+                    title: t("total"),
+                    value: numFormat(
+                      timeseries_callout.data.data[currentState][key].cumul.value,
+                      "standard"
+                    ),
+                  },
+                ]}
+              />
+            ))}
+          </div>
```

Comment:
Wrap all timeseries under `SliderProvider` and pass the `enableAnimation` prop. Also the padding between the slider and the row of timeseries looks smaller than in figma